### DISCUSSION TITLE
【400750】战争号令：地狱之门 Call to Arms - Gates of Hell: Ostfront

### DISCUSSION THREAD

**Discussion by gonghaocheng-chia** [pjy612/SteamManifestCache#271] [Share]
`AppID：400750`
keys
```
400751;69217ebc13da662b0ca67091b3077114ca5cff0814e2a319bc8f998462bed8d6
400757;66633111fcabc113cdc23e2b086d88841a6e535b8a4a8b10c812d5dbab10e344
```
code：D86FC50C37C5B06B
邮箱：1903723767@qq.com
[400750_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13827615/400750_config.vdf.txt)

**Comment by pjy612** [comment]
1. 密钥未关联 depotid
2. 附件上传未完成

**Reply by gonghaocheng-chia** [reply]
已更新

**Reply by pjy612** [reply]
中文语言包 的密钥不写？
https://steamdb.info/app/400750/depots/

**Reply by gonghaocheng-chia** [reply]
已添加
